### PR TITLE
Update JAX landing page - Flax

### PR DESCRIPTION
**Pull request by 8bitmp3**
@jakevdp

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -170,11 +169,10 @@ maintains an up-to-date list.
...
-.. _NNX: https://flax.readthedocs.io/en/latest/nnx/
 .. _Numpyro: https://num.pyro.ai/en/latest/index.html
 .. _Optax: https://optax.readthedocs.io/
 .. _Optimistix: https://github.com/patrick-kidger/optimistix
 .. _Orbax: https://orbax.readthedocs.io/
 .. _PyMC: https://www.pymc.io/
-.. _Tensorflow datasets: https://www.tensorflow.org/datasets
-.. _Tensorflow probabilty: https://www.tensorflow.org/probability
+.. _TensorFlow Datasets: https://www.tensorflow.org/datasets
+.. _TensorFlow Probabilty: https://www.tensorflow.org/probability
```
Comment (jakevdp) [inline docs/index.rst:178]: If your intent is to change how these appear on the page, you'll also have to change the places where they are referenced in the text above.

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (8bitmp3) [reply docs/index.rst:178]: On it.

Comment (8bitmp3) [reply docs/index.rst:178]: Done @jakevdp

**Review by jakevdp — APPROVED** [review]
Thanks!

**Pull request merged**